craig-rolleston/week4/fri/orm-travel-site/views/attractions_show.erb

```diff
@@ -0,0 +1,19 @@
+<h2>
+    <%= @attractions.travel.location if @attractions.travel%>
+</h2>
+
+<a  href="/attractions/<%=@attractions.id%>/edit">Edit</a>
+|
+<a href="/attractions/<%=@attractions.id%>/delete">Delete</a>
+
+<p>
+    <strong>Name:</strong>
+    <%= @attractions.name %>
+</p>
+
+<p>
+    <strong>Destination:</strong>
+    <%= @attractions.travel.location if @attractions.travel%>
+</p>
```

Comment:
good work, but you need to make the associative link!

you need to wrap this in an `<a>` tag and the link would be to the `travel.id`